index.md

```diff
@@ -188,7 +188,7 @@ title: Distributed SQL query engine for big data
     <div class="card mb-4">
       <div class="card-body text-center">
         <h3 class="card-header-title mb-3">Reading material</h3>
-        <p>Get a digital copy of <a href="{{ site.url }}{% post_url 2020-04-11-the-definitive-guide %}">the definitive guide about the Trino
```

Comment:
Why not use the absolute link? `{{ site.url }}`

Reply:
its longer in the source and doesnt work any better

Reply:
I think we should probably clean out the whole site around this aspect at some stage.. maybe I will send follow up PRs.